bin/reth/src/optimism.rs

```diff
@@ -28,9 +29,8 @@ fn main() {
             .extend_rpc_modules(move |ctx| {
                 // register sequencer tx forwarder
                 if let Some(sequencer_http) = rollup_args.sequencer_http {
-                    ctx.registry.set_eth_raw_transaction_forwarder(Arc::new(SequencerClient::new(
-                        sequencer_http,
-                    )));
+                    ctx.registry
+                        .set_sequencer_client(Arc::new(SequencerClient::new(sequencer_http)));
```

Comment:
@emhane I believe we can integrate this with the OP node addons so we don't need the function on the registry anymore

https://github.com/paradigmxyz/reth/blob/5a9de035bf779606247b3e67ae1907011e2339b7/crates/optimism/node/src/node.rs#L103-L105

and roll our own provider here

https://github.com/paradigmxyz/reth/blob/5a9de035bf779606247b3e67ae1907011e2339b7/crates/optimism/rpc/src/eth/mod.rs#L176-L180

Reply:
since https://github.com/paradigmxyz/reth/pull/9879 we can set the sequencer in the `OpEthApi` builder function, and the `UpdateRawTxForwarder` trait can be removed